.github/workflows/sync_crowdin.yml

```diff
@@ -1,12 +1,14 @@
-name: Crowdin Action
+name: Sync Crowdin
 
 on:
   workflow_dispatch:
   push:
-    branches: [dev]
+    branches:
+      - dev
```

Comment:
@Ushie  This is slightly problematic, the workflow shouldn't run on every commit, maybe there's a better trigger?

Reply:
In my opinion the workflow should run when we dispatched it or let it automatically run on every week/month

Reply:
A manual trigger or CRON would work fine, but I feel like those can be easily forgotten

There's no reminder when using a manual trigger, and the CRON job wouldn't be likely to run during the time when someone is working on the project, by running it when pushing to dev you can be reminded by the "Recent pushes" notification

Reply:
I am saying that the trigger may be reduced to the JSON file, so when the JSON file is changed, the trigger runs.